index.js

```diff
@@ -12,8 +15,32 @@ formatCustomerBalance("Jane Doe", 250); // => "John Doe has a balance of 250 KD.
 
 function getBooksPublishedAfter2010(books) {
   // Write code here
+  // const book = getBooksPublishedAfter2010.fitler()
+  // books[1].year >= 2012;
+  // return books[1];
+  // return (books) => books.year >= 2010;
+  // return (books = books.year >= 2010);
+  books.forEach((books) => {
+    console.log(books.year);
+  });
 }
 
+// movies.forEach((movies) => {
+//   console.log(movies.title);
+// });
+// console.log(updatesMovies(movies));
+// console.log(getBooksPublishedAfter2010[2]);
+// console.log(getBooksPublishedAfter2010[books]);
+// const newbook = getBooksPublishedAfter2010.((books) => books.year >= 2010);
+
+// console.log(newbook);
+
+// const getBooks2010 = getBooksPublishedAfter2010.filter(
```

Comment:
This code has been copied from one of your tasks files, btw this is the correct answer to the questions.